src/components/structures/BottomLeftMenu.js

```diff
@@ -98,9 +112,21 @@ module.exports = React.createClass({
 
     render: function() {
         var TintableSvg = sdk.getComponent('elements.TintableSvg');
+
+        var homeButton;
+        if (window.localStorage.getItem('mx_team_token')) {
```

Comment:
Might be nice if this loaded the team token at mount time & then used the state here, which wouldn't hit localstorage on every render (and makes render more a function of the state).